src/OpenLoco/src/Paint/Paint.cpp

```diff
@@ -990,6 +990,7 @@ namespace OpenLoco::Paint
             case Ui::ViewportInteraction::InteractionItem::roadStation:
             case Ui::ViewportInteraction::InteractionItem::road:
             case Ui::ViewportInteraction::InteractionItem::roadExtra:
+            case Ui::ViewportInteraction::InteractionItem::bridge:
```

Comment:
The problem with bridges is we don't know if they belong to roads or tracks. As such this change will make tracks also lose their bridge when you turn off roads.

Reply:
With the last commit I removed this and removed see through bridges from see through tracks. I also moved airports and docks to buildings for see through purposes